### PR TITLE
Use CHAR for GUID and UUID types in MySQL

### PR DESCRIPTION
**Pull request by aliaspooryorik**
Use CHAR for GUID and UUID types in MySQL

### CODE REVIEW

File: models/Grammars/BaseGrammar.cfc [hunk 1]
```diff
@@ -1483,7 +1483,7 @@ component displayname="Grammar" accessors="true" singleton {
     }
 
     function typeUUID( column ) {
-        return typeChar( column, 35 );
+        return typeChar( column );
```
Comment (jclausen) [inline models/Grammars/BaseGrammar.cfc:1500]: Why did you remove the lengths from these two return types, @aliaspooryorik ?  Those lengths are the specification lengths for GUID (36) and UUID (35)

Comment (aliaspooryorik) [reply models/Grammars/BaseGrammar.cfc:1500]: Because the 2nd argument is not referenced anywhere @jclausen. See https://github.com/coldbox-modules/qb/blob/main/models/Grammars/BaseGrammar.cfc#L1432
```
function typeChar( column ) {
        return "CHAR(#column.getLength()#)";
}
```

Comment (aliaspooryorik) [reply models/Grammars/BaseGrammar.cfc:1500]: Bump @jclausen 
Note that the testcases cover the field length and are passing
tests/specs/Schema/MySQLSchemaBuilderSpec.cfc
